File: java/Unirest.java
HttpResponse<String> response = Unirest.post("https://api.neginpardakht.ir/v1/transaction/request")
  .header("Content-Type", "application/json")
  .header("AuthToken", "U2FsdGVkX19uc/H/k4C/...IW3gyPy9iQ==")
  .body("{\n\t\"merchantId\": \"67f51bc64fb...b619\",\n\t\"amount\": 20000,\n\t\"callbackUrl\": \"https://webya.ir\"\n}")
  .asString();

File: nodejs/unirest.js
const unirest = require("unirest");

const req = unirest("POST", "https://api.neginpardakht.ir/v1/transaction/request");

const CookieJar = unirest.jar();
req.jar(CookieJar);

req.headers({
  "Content-Type": "application/json",
  "AuthToken": "U2FsdGVkX19uc/H/k4C/e6oe5Y...y9iQ=="
});

req.type("json");
req.send({
  "merchantId": "67f51b...619",
  "amount": 20000,
  "callbackUrl": "https://webya.ir"
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

